### DISCUSSION TITLE
show the alert threshold in the graph

### DISCUSSION THREAD

**Discussion by wederbrand** [grafana/grafana#56537] [Alerting]
In older versions of Grafana, with the old alert manager, the alert threshold was visible, and settable, directly from the graph. 

Please make it possible to show and control the threshold directly in the graph.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
